### ISSUE TITLE
Change password dosen't make any sense for seciurity

### ISSUE THREAD

**Issue by KonoromiHimaries** [rfjakob/gocryptfs#766]
if i copy gocryptfs.diriv and gocryptfs.conf
next create a single file with random text inside and paste it to mount folder
next change a password with '-passwd' option
next change a text inside this file
next unmount and copy back gocryptfs.diriv and gocryptfs.conf that were copied before
then i have new text inside this file with old password

**Comment by jumoog** [comment]
It's not a bug see https://github.com/rfjakob/gocryptfs/blob/master/Documentation/MANPAGE.md#-passwd

**Comment by jumoog** [comment]
You only change the password. The master key is still the same as before. And only the master key is used for encryption and decryption.

**Comment by jumoog** [comment]
This is always the case if you use e.g. LUKS to encrypt your hard disk:
- make a backup of the headers
- change the password
- restore the old headers
Now you can also use the old password.

**Comment by KonoromiHimaries** [comment]
Maybe create sub-key for the new password and use it at replace master-key for new changes in files.

**Comment by rfjakob** [comment]
I agree that it's somewhat stupid, but as mentiones, basically all disk
encryption works like this.

I will at least add a warning into the man page.

The use case for changing the password is to change it to something very
strong *before* you upload it somewhere.
